share/qtcreator/sfdk/modules/20-building-mb2.json

```diff
@@ -163,5 +168,34 @@
             "trArgument": "<file>",
             "trDescription": "In case a passphrase-protected signing key is selected for signing packages, read the passphrase from the <file>. Only the first line will be read."
         }
+    ],
+
+    "hooks":
+    [
+        {
+            "name": "pre-pull-build-requires",
+            "trSynopsis": "[<dependency>...]",
+            "trDescription": "This hook is invoked by commands that take care of updating build time dependencies – prior to doing so.\n\nThe arguments and available environment variables are the same as with the 'pull-build-requires' hook."
```

Comment:
Consider e.g. "This hook is invoked by commands that take care of updating build time dependencies, before performing the update or install of the dependencies."

Reply:
hm, here the first sentence would use the word update in the general sense of getting the requirements fulfilled (somehow), while the second sentence would use it in the specific sense of updating packages that fulfill the requirements. I would rather leave it as is - it should be clear enough and should there be any doubt, the "build-requires pull" command states more details.

Reply:
Well it's not clear to me what the "doing so" refers to.

Reply:
It refers to "updating build time dependencies". Does this help? https://ludwig.guru/s/prior+to+doing+so

Reply:
Perhaps you would like to add that link to the description then, to help make it clear what you mean?